composer.json

```diff
@@ -36,6 +36,7 @@
         "doctrine/cache": "^2.0",
         "doctrine/doctrine-bundle": "^2.3",
         "doctrine/orm": "^2.7",
+        "egulias/email-validator": "^3.2",
```

Comment:
is this needed for 8.2 ?

Reply:
https://github.com/PrestaShop/PrestaShop/pull/29116#issuecomment-1191285154

Reply:
Removed